src/pokemon.c

```diff
@@ -1993,25 +1992,27 @@ u16 MonTryLearningNewMove(struct Pokemon *mon, bool8 firstMove)
     if (firstMove)
     {
         sLearningMoveTableID = 0;
-
```

Comment:
Any reason you're removing these newlines?

Reply:
[`4bcb1c5` (#4901)](https://github.com/rh-hideout/pokeemerald-expansion/pull/4901/commits/4bcb1c54b1d9f66fab224d001de5116b1f09fc9b)